share/functions/__fish_paginate.fish

```diff
@@ -5,7 +5,7 @@ function __fish_paginate -d "Paginate the current command using the users defaul
         set cmd $PAGER
     end
 
-    if test -z (commandline -j)
+    if test -z (string join '' (commandline -j))
```

Comment:
I'd recommend piping into string, because it's a little shorter, easier to follow (left-to-right flow) and because it won't fail if the job starts with "-" (mostly theoretical, but still).

Reply:
`commandline -j | string join ''`

Reply:
yeah, makes sense